deposit/bytecode-verification/deposit-spec.ini

```diff
@@ -55,55 +55,55 @@ VYPER_GENERATED_BOUNDS:
     [ 128 := #buf(32, 1701411834604692317316873037158841057270000000000) ]
     [ 160 := #buf(32, 115792089237316195423570985006986496018665292348323691002298742950633129639936) ]
 LOCAL_MEM_UPDATE_BY_TO_LITTLE_ENDIAN_64_DEPOSIT_COUNT:
-    [ 352 := #buf(32, {RETURN_ADDR_FROM_TO_LITTLE_ENDIAN_64}) ]  /* return address */
-    [ 320 := #buf(32, DEPOSIT_COUNT) ]  /* argument */
-    [ 384 := #buf(32, Y8) ]             /* y: uint256 = (y << 8) + bitwise_and(x, 255) */
-    [ 416 := #buf(32, X8) ]             /* x: uint256 = (x >> 8) */
-    [ 448 := #buf(32, 8) ]              /* for-loop index */
-    [ 480 := #buf(32, X7 &Int 255) ]    /* bitwise_and(x, 255) */
-    [ 544 := #buf(32, Y8) ]             /* memcpy(544 <- 384, 32) */
-    [ 536 := #buf(32, 8) ]              /* for slice(convert(y, bytes32), start=24, len=8) */
-    [ 704 := #buf(32, 8) ++ #bufSeg(#buf(32, Y8), 24, 8) ]  /* return: bytes[8] */ /* memcpy(704 <- 536, 40) */
+    [ 352 /* = 320 + 32 *  1      */ := #buf(32, {RETURN_ADDR_FROM_TO_LITTLE_ENDIAN_64}) ]  /* return address */
+    [ 320 /* = 320 + 32 *  0      */ := #buf(32, DEPOSIT_COUNT) ]  /* argument */
```

Comment:
any reason why these are out of order?

Reply:
The main reason is that K does not yet fully support the AC matching over maps.  So, I put the map update in the order they appear in the execution of the bytecode.  The advantage of this order is that it clearly shows how the local memory is updated in each step of the bytecode execution.